js/Events/logInEvents.js

```diff
@@ -0,0 +1,79 @@
+interviewApp.Events.EventsLogIn = function () {
+    const submitFormListener = function (event) {
+        const alert = document.querySelector('.alert');
+        if (alert) {
+            event.target.removeChild(alert);
+        }
+
+        event.stopPropagation();
```

Comment:
Can you describe to me what caused the need for the event propagation to be interrupted?